blocks/library/list/index.js

```diff
@@ -72,6 +112,17 @@ registerBlock( 'core/list', {
 		return (
 			<Editable
 				tagName={ nodeName.toLowerCase() }
+				onConfig={ ( settings ) => ( {
+					...settings,
+					plugins: ( settings.plugins || [] ).concat( 'lists' ),
+					lists_indent_on_tab: false,
+				} ) }
+				onSetup={ ( editor ) => {
+					editor.on( 'nodeChange', ( nodeInfo ) => {
+						setAttributes( { internalListType: findInternalListType( nodeInfo ) } );
+					} );
+					setAttributes( { editor } );
```

Comment:
I'll leave it for now.

I was unaware that state was being serialized beyond what I was outputting in save. I think it should be opt-in.